packages/ag-charts-community/src/dom/focusIndicator.ts

```diff
@@ -36,7 +54,7 @@ export class FocusIndicator {
             this.path.setAttribute('d', bounds.svgPathData(transform));
             this.show(this.svg);
         } else {
-            setElementBBox(this.div, bounds);
+            setElementBBox(this.div, bounds.relativeClip(hoverRect));
```

Comment:
Change this to `BBoxValues.relativeClip(bounds, hoverRect)` to avoid the unnecessary `BBox` type requirement.